### PR TITLE
Fix display in devportal switch

### PR DESCRIPTION
**Pull request by Krishanx92**

### CODE REVIEW

**Comment by Krishanx92** [comment]
Fix https://github.com/wso2/product-apim/issues/10332
Fix https://github.com/wso2/product-apim/issues/10187

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DisplayDevportal.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            if (api.apiType === API.CONSTS.APIProduct) {
+                restAPIProduct.displayInDevportalProduct(api.id, base64url.encode(event.target.name), body);
+            } else {
+                restApi.displayInDevportalAPI(api.id, base64url.encode(event.target.name), body);
+            }
+        }
+    };
+
+    return (
+        <>
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DisplayDevportal.jsx:63]: We don't need a [Fragment](https://reactjs.org/docs/fragments.html) here, right?

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DisplayDevportal.jsx:63]: > A common pattern in React is for a component to return multiple elements. Fragments let you group a list of children without adding extra nodes to the DOM.

Here it's only one child

**Review by tmkasun — APPROVED** [review]

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DisplayDevportal.jsx (hunk 1 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DisplayDevportal.jsx]: Yes, Removed

**Pull request merged**